.buildkite/pipeline.yml

```diff
@@ -358,6 +144,10 @@ steps:
       docker-compose#v3.3.0:
         run: cocoa-maze-runner
         command:
+          - "features/app_and_device_attributes.feature"
```

Comment:
Assuming this is a "barebones" test run, it should use only `features/barebone_tests.feature`

Reply:
Changed that over in the shortest test runs.